redash_toolbelt/examples/migrate.py

```diff
@@ -0,0 +1,760 @@
+"""
+Author: Jesse Whitehouse
+Last updated: 24 August 2021
```

Comment:
```suggestion
Last updated: 1 September 2021
```

Didn't catch this while updating earlier today 🤦